controller.js

```diff
@@ -1,30 +1,41 @@
 
-const returnTopics = require("./model")
+const {returnTopics,returnArticle} = require("./model")
 const fs = require('fs')
-const data = require('./endpoints.json')
+const endpointsData = require('./endpoints.json')
 
 
 
 function getTopics(req, res, next){
-    
-returnTopics()
-.then(data => {
-    
+    returnTopics()
+    .then(data => {
     res.status(200).send({'topics': data})
-})
-.catch(next)
+    })
+    .catch(next)
+    }
+
+function getArticle(req, res, next){
+    const articleId =req.params.article_id
+    console.log(articleId)
+    if(isNaN(articleId))
```

Comment:
We don't need to handle this error with a custom response, in the first instance it's usually best to allow the error to pass to the sql query and see if sql will throw an error first. Only if it doesn't and we want it to be an error, do we then need to consider a custom error handler and bring in a promise.reject that our catch can grab.